lib/Value/String.pm

```diff
@@ -80,7 +80,16 @@ sub compare {
 #
 #  Mark a string to be display verbatim
 #
-sub verb {shift; return "\\verb".chr(0x1F).(shift).chr(0x1F)}
+sub verb {
+  shift;
+  my $s = shift;
+  $s =~ 's/\r/ /g';
```

Comment:
I don't think the single quotes should be there.  Does that actually work to perform a substitution?

Reply:
When I do command line perl substitutions, I use single quotes and it works. Like `perl -pi -e 's/\r/ /g' *.txt`. But I will test (and change to double if it's more standard here.)

Reply:
The quotes here are shell characters that are not part of the actual perl code.  The line should be

    $s =~ s/\r/ /g;

with no quotation marks of any kind.